wit/client.go

```diff
@@ -128,3 +128,31 @@ func GetSpace(ctx context.Context, client *api.Client, spaceID uuid.UUID) (*api.
 	}
 	return client.DecodeSpaceSingle(resp)
 }
+
+func GetSpaces(ctx context.Context, client *api.Client, spaceIDs []uuid.UUID) ([]*api.SpaceSingle, error) {
+	var spaces []*api.SpaceSingle
+	for _, spaceID := range spaceIDs {
+		space, err := GetSpace(ctx, client, spaceID)
+		if err != nil {
+			return nil, err
+		}
+		spaces = append(spaces, space)
+	}
+	return spaces, nil
+}
+
+func GetCodebases(ctx context.Context, client *api.Client, url string) (*api.CodebaseList, error) {
```

Comment:
Add a couple of tests for this, please.

Reply:
end-to-end integration invokes this and in case any bug it will catch here.  other than that I don't see this is doing much than making http request and returning response.